### PR TITLE
chore: Updates underscore within Courage

### PR DESCRIPTION
**Pull request by jmelberg-okta**
## Description:

- Updates Okta's internal Courage framework version to utilize `underscore@1.13.1`.

## PR Checklist

- [x] Have you verified the basic functionality for this change?

### Reviewers:

- @rajaguruduraisamy-okta 
- @aarongranick-okta 

### Issue:

- [OKTA-404358](https://oktainc.atlassian.net/browse/OKTA-404358)

### CODE REVIEW

File: packages/@okta/courage-for-signin-widget/yarn.lock [hunk 1]
```diff
@@ -3574,10 +3569,10 @@ uglifyjs-webpack-plugin@^0.4.6:
...
-underscore@1.8.3:
-  version "1.8.3"
-  resolved "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta-master/underscore/-/underscore-1.8.3.tgz#4f3fb53b106e6097fcf9cb4109f2a5e9bdfa5022"
-  integrity sha1-Tz+1OxBuYJf8+ctBCfKl6b36UCI=
+underscore@1.13.1:
+  version "1.13.1"
+  resolved "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta-master/underscore/-/underscore-1.13.1.tgz#0c1c6bd2df54b6b69f2314066d65b6cde6fcf9d1"
+  integrity sha1-DBxr0t9UtrafIxQGbWW2zeb8+dE=
 
 underscore@>=1.8.3:
```
Comment (rajaguruduraisamy-okta) [inline packages/@okta/courage-for-signin-widget/yarn.lock:3577]: we can get rid of old version of underscore by remove the courage dependency and readding.

Comment (jmelberg-okta) [reply packages/@okta/courage-for-signin-widget/yarn.lock]: Good callout. This is going to modify a few other dependency versions, but _should_ be OK.

File: packages/@okta/courage-for-signin-widget/webpack.config.js [hunk 2]
```diff
@@ -3,7 +3,6 @@ const { resolve } = require('path');
 const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
 const { BannerPlugin } = require('webpack');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
-const ReplaceInFileWebpackPlugin = require('replace-in-file-webpack-plugin');
```
Comment (nikhilvenkatraman-okta) [inline packages/@okta/courage-for-signin-widget/webpack.config.js:6]: What is the reason to remove this ?

**Review by nikhilvenkatraman-okta — APPROVED** [review]
Looks good. Just left a question.

File: packages/@okta/courage-for-signin-widget/webpack.config.js (hunk 2 above)
Comment (jmelberg-okta) [reply packages/@okta/courage-for-signin-widget/webpack.config.js:6]: We used this dependency as a workaround for removing a deprecated API from the Courage bundle. Now that Courage uses the updated API - we can safely remove it 😄 

See [this comment](https://github.com/okta/okta-signin-widget/pull/2048/files#diff-eec33d248efcfb44568c1072566a82e05989edd169d3635238f2c1ee52e6ad9dL121-L123) for more details.

**Pull request closed without merging**